### PR TITLE
Fix path.join bug; label extraction completed; added LSTM prediction …

### PR DESCRIPTION
**Pull request by mabhijithn**
…frequency as a variable

### CODE REVIEW

**Comment by mabhijithn** [comment]
Fixes the two issues posted [label extraction missing](https://github.com/mabhijithn/irregulars-neureka-codebase/issues/4) and a [typo](https://github.com/mabhijithn/irregulars-neureka-codebase/issues/4)

File: training/4-train-lstm.py [hunk 1]
```diff
@@ -47,8 +51,14 @@ def prepare_file(file_i, filename, classifiers, f_nick, model_type):
     x = np.array(x)
     x = np.transpose(x)
     if model_type == 'lstm' or model_type == 'gru':
-        x = x.reshape((len(x), 1, len(x[0])))    
-    return x
+        x = x.reshape((len(x), 1, len(x[0])))
+    # Collect the true lables
+    seizures = nedc.loadTSE(os.path.join(EDF_ROOT,filename[:-4]+'.tse'))
+    
+    # Create labels at 1Hz sampling rate
```
Comment (jon-dan) [inline training/4-train-lstm.py]: Comment seems misleading as y will be labels at fs sampling rate

File: training/4-train-lstm.py [hunk 2]
```diff
@@ -28,12 +32,12 @@
...
 def load_filenames():
     filenames = list()
-    with h5py.File(os.joind(PREDICTION_ROOT, 'prediction_test_iclabel.h5'), 'r') as f:
+    with h5py.File(os.path.join(PREDICTION_ROOT, 'prediction_test_iclabel.h5'), 'r') as f:
         filenames = list(f['filenames'])
     return filenames
 
 
-def prepare_file(file_i, filename, classifiers, f_nick, model_type):    
+def prepare_file(file_i, filename, classifiers, f_nick, model_type, fs):
```
Comment (jon-dan) [inline training/4-train-lstm.py]: sure I am guilty of the same but I don't like functions without description a docstring (do you what you want of comment)

**Review by jon-dan — COMMENTED** [review]
Looks good, would just change the comment about generating labels at 1Hz to generating labels at fs as you are leaving this free to user of function

File: training/4-train-lstm.py (hunk 2 above)
Comment (mabhijithn) [reply training/4-train-lstm.py]: Mostly fixed with 70ae9d6b5ae88bdf72495cb94c61bd085a904961

File: training/4-train-lstm.py (hunk 1 above)
Comment (mabhijithn) [reply training/4-train-lstm.py]: Fixed: d845aa7b50db125860ab9e794f8f23d927711717

**Pull request merged**